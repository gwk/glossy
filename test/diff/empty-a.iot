{
  'args': ['test/empty.txt', 'test/a.txt'],
}
