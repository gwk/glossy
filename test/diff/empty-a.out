pat v0
test/empty.txt

$
+ a
